hazelcast/src/main/java/com/hazelcast/monitor/LocalFlakeIdGeneratorStats.java

```diff
@@ -0,0 +1,30 @@
+/*
+ * Copyright (c) 2008-2018, Hazelcast, Inc. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.hazelcast.monitor;
+
+/**
+ * Local flake id statistics interface to be used
+ * by {@link com.hazelcast.monitor.MemberState} implementations.
+ */
+public interface LocalFlakeIdGeneratorStats extends LocalInstanceStats {
+
+    /**
+     * @return the total number of times the ID generator has been used to generate
+     *         a new ID batch since its {@link #getCreationTime()}.
+     */
+    long getUsageCount();
```

Comment:
This should be renamed to `getIdBatchCount`, this is too generic, esp. if we add other stats in the future.

Reply:
I found your suggestion over-specific as it refers to implementation details that should not matter for a end user or for statistics shown to a end user so I kept the name.

Reply:
I don't agree. That detail is already mentioned in javadoc. I would be puzzled what does it mean to have "100k usages". 
"100k ID batches" would be much better.
@mmedenjak, wdyt?